src/game/Tactical/LoadSaveObjectType.cc

```diff
@@ -38,7 +38,7 @@ void ExtractObject(DataReader& d, OBJECTTYPE* const o)
 			EXTR_U8(d, o->ubGunAmmoType)
 			EXTR_U8(d, o->ubGunShotsLeft)
 			EXTR_SKIP(d, 1)
-			EXTR_U16(d, o->usGunAmmoItem)
+			o->usGunAmmoItem = d.read<ItemId>();
```

Comment:
You're doing this often enough that adding EXTR_ItemId and INJ_ItemId macros are probably worthwhile